src/FA2.hs

```diff
@@ -0,0 +1,85 @@
+module FA2 
```

Comment:
I know it's WIP and you are experimenting, but note that eventually it should be incorporated into the contract that we have. Currently the module is called `Basic`, we need something like `Lorentz.Contracts.BaseDAO` instead.

Reply:
> Can you please estimate how much you need to finish that?

I think It won't take long, since token_metadata_registry seems finished (with one mistake, that will be corrected now) and updateOperators also will be finished when addOperator is finished. 

> For other methods you can do as you wish:

I think it will be better if this two stay as pure Lorentz functions, because they are near-complete. All other stuff will be transported as indigo functions firstly, and then decopypasted

Reply:
I don't understand you. Is it Fine to store it like `Lorentz/Contracts/BaseDAO/FA2.hs`?

Reply:
It's fine to store FA2 logic in `Lorentz/Contracts/BaseDAO/FA2.hs`. But there should be the root module that exports the whole contract. This module should be called `Lorentz.Contracts.BaseDAO` and should export `Contract Parameter Storage`. This contract, in turn, should replace the `measureBoaConstrictor` contract in `Main.hs`.